### PR TITLE
Add source to updates_generator before continuing.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
Follow up from #65 which resolves the problem of source packages not being synchronized.

Thanks to @clalancette for reporting the issue.

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
Once this PR merges I think we can restore the testing repository by manually triggering a sync to testing for any affected rosdistros/architectures.

File: scripts/sync_ros_packages.py [hunk 1]
```diff
@@ -94,6 +94,13 @@
 
         updates_generator.add_update_element(conf.UpdateElement(**d))
 
+        # The source architecture doesn't provide a Packages file to diff with.
+        # But it should still be added to the updates_generator above so the
+        # source repository is updated.
+        if arch == 'source':
```
Comment (clalancette) [inline scripts/sync_ros_packages.py]: This looks OK, but you also need to remove the old check at the top of the loop, I believe.

Comment (nuclearsandwich) [reply scripts/sync_ros_packages.py]: :man_facepalming:

Comment (nuclearsandwich) [reply scripts/sync_ros_packages.py]: Fixed via force push of my (apparently) uncommitted changes.

**Comment by nuclearsandwich** [comment]
Repositories that need a sync in order to fix testing:
* [ ] Indigo ARM
* [ ] Indigo Default
* [ ] Kinetic Default
* [ ] Kinetic UXHF
* [ ] Kinetic UXV8
* [ ] Melodic Default
* [ ] Melodic DS
* [ ] Melodic DSV8
* [ ] Melodic UBHF
* [ ] Melodic UBV8

**Pull request merged**